traitsui/qt4/video_editor.py

```diff
@@ -114,16 +217,17 @@ def init(self, parent):
         parent : QWidget or None
             The parent widget for this widget.
         """
-
-        self.control = QVideoWidget()
```

Comment:
I think that the default use case should probably just use `QVideoWidget` particularly if the alternative fails in some Qt APIs.  Something like: if we have an image manipulation function on the factory then instantiate an `ImageWidget` which uses that manipulation function; otherwise create a standard `VideoWidget` as the control.

Possibly need to do something about the surface as well.

Reply:
Should be implemented now.